### PR TITLE
Buildable Wall Items 1

### PR DESCRIPTION
**Pull request by SinTwo**
Newscasters, Status Displays, AI Status Displays, Entertainment Consoles, Fire Extinguisher Cabinets, and Notice Boards can now be built.

### CODE REVIEW

File: code/modules/materials/material_recipes.dm [hunk 1]
```diff
@@ -110,6 +112,7 @@
 	recipes += new/datum/stack_recipe("beehive assembly", /obj/item/beehive_assembly, 4)
 	recipes += new/datum/stack_recipe("beehive frame", /obj/item/honey_frame, 1)
 	recipes += new/datum/stack_recipe("book shelf", /obj/structure/bookcase, 5, time = 15, one_per_turf = 1, on_floor = 1)
+	recipes += new/datum/stack_recipe("book shelf", /obj/item/frame/noticeboard, 4, time = 5, on_floor = 1)
```
Comment (Neerti) [inline code/modules/materials/material_recipes.dm]: I think you forgot to change the recipe name here.

**Comment by SinTwo** [comment]
Def' did not forget that Neerti... I don't know what you are talking about....

**Comment by comma** [comment]
Looks like PDA sprites commit crawled in here too

**Comment by SinTwo** [comment]
@comma Commit to fix that SHOULD be in there.

**Comment by Zuhayr** [comment]
Conflicts.

**Comment by Zuhayr** [comment]
I do not personally like the amount of boilerplate code in this pull, but Sin has said that it's a preliminary rework for a proper implementation. Not sure I am in favour of this being merged due to the tendency for interim systems to become permanent, so I will abstain from :+1: or :-1:.

**Comment by SinTwo** [comment]
At time of writing this, unified frame system is currently working with both Computers and Machines. Slight issues with Alarm and Displays but working on that now.

**Pull request closed without merging**